invisible_cities/cities/components.py

```diff
@@ -503,6 +503,7 @@ def cdst_from_files(paths: List[str]) -> Iterator[Dict[str,Union[pd.DataFrame, M
         try:
             cdst_df    = load_dst (path,   'CHITS', 'lowTh')
             summary_df = load_dst (path, 'Summary', 'Events')
+            kdst_df    = load_dst (path,     'DST', 'Events', ignore_errors=True)
         except tb.exceptions.NoSuchNodeError:
             continue
```

Comment:
Is this try/except clause for a specific line or for both cdst and summary?
If it is for just one of them, then that line should be the only one in the try block. If it is for both of them it's fine.
The new line (kdst_df) should definitely go outside of the try block because it is not supposed to raise a `NoSuchNodeError`.

Reply:
Both `cdst_df` and `summary_df` should be inside. Ok, it is more correct to move the `kdst_df` outside.